README.md

```diff
@@ -2,6 +2,8 @@
 
 ## Summary
 
+This project aims to build an R and Python package that elegently performs data pre-processing in a fast and easy manner.With four separate functions that will come along with the b4prep package, users will have greater flexibility in handling many different types of datasets in the wild or those collected by them. With b4prep package, users will be able to smoothly pre-process their data and have it ready for the machine learning model of their choice.
```

Comment:
- did we change the package name to b4prep? or is it still b4model?
- besides that, i really like this summary!

Reply:
we changed the name to rb4model, I will correct it. Thanks for picking it up Andrea.

Reply:
Maybe you can merge this one, and I will send another PR with correction.

Reply:
No need to merge this PR first and send another PR. This is because the new PR you send will automatically update this PR!